terra-app-helm/aou-rstudio-chart/test_values.yaml

```diff
@@ -0,0 +1,66 @@
+# Provided by the app descriptor
+image:
+  image: "gcr.io/cryptic-skyline-320001/terra-rstudio-aou:smoke-test-1"
+
+# Provided by Leonardo
+configs: {}
+# WORKSPACE_NAME: my-ws
+
+# Provided by Leonardo
+extraEnv:
+# - name: WORKSPACE_NAME
+#   value: my-workspace
+
+imagePullSecrets: []
+nameOverride: aou-rstudio
+fullnameOverride: aou-rstudio
+
+# Provided by Leonardo
+serviceAccount:
+  name: "test-service-account"
+
+
+service:
+  type: ClusterIP
+  port: 80
+
+# TLS and rewrite rules provided by Leonardo
+ingress:
+  enabled: true
+  annotations:
+#   nginx.ingress.kubernetes.io/auth-tls-pass-certificate-to-upstream: "true"
+#   nginx.ingress.kubernetes.io/auth-tls-secret: "{{ $.Release.Namespace }}/ca-secret"
+#   nginx.ingress.kubernetes.io/auth-tls-verify-client: "on"
+#   nginx.ingress.kubernetes.io/auth-tls-verify-depth: "1"
+#   nginx.ingress.kubernetes.io/ssl-redirect: "true"
+#   nginx.ingress.kubernetes.io/proxy-redirect-from: "https://k8s-host/"
+#   nginx.ingress.kubernetes.io/proxy-redirect-to: "https://leo-host/"
+   nginx.ingress.kubernetes.io/rewrite-target: "/$2"
+   nginx.ingress.kubernetes.io/proxy-redirect-from: "$scheme://$host/"
+   nginx.ingress.kubernetes.io/proxy-redirect-to: "$scheme://$host/rstudio/"
+
+# Provided by Leonardo
+nodeSelector: {}
+# cloud.google.com/gke-nodepool: "nodepool-name"
+
+# either GCE PD provided by Leonardo; or hostPath provided by terra-docker-local.sh
+persistence:
+  size: 100G
+  gcePersistentDisk: "disk--test"
+
+welder:
+  extraEnv:
+  - name: GOOGLE_PROJECT
+    value: "cryptic-skyline-320001"
+  - name: STAGING_BUCKET
+    value: "aou-startup-script"
+  - name: CLUSTER_NAME
+    value: "all-of-us-123"
+  - name: RUNTIME_NAME
```

Comment:
Oh I am curious to see if that was actually needed? We won't be spinning up a runtime from leo